llvm/lib/Target/AMDGPU/VOPInstructions.td

```diff
@@ -1484,8 +1484,8 @@ multiclass VOP3_Real_dpp8_with_name<GFXGen Gen, bits<10> op, string opName,
   let AsmString = asmName # ps.Pfl.AsmVOP3DPP8,
       DecoderNamespace = "DPP8"#Gen.DecoderNamespace#
                          !if(ps.Pfl.IsRealTrue16, "", "_FAKE16"),
-      OtherPredicates = !if(ps.Pfl.IsRealTrue16, [UseRealTrue16Insts],
```

Comment:
There are other places in the same file that also need to be taken care of:
VOP3_DPP16_Gen, VOP3_Real_Gen for example.
Do we actually have to introduce True16Predicate? 
I think we have already used listconcat to add a new predicate into OtherPredicates list.

Reply:
Pasting `OtherPredicates` could help your particular case, but wouldn't resolve the more general problem of overriding predicates for various groups of instructions and inheriting them from their corresponding pseudos.

The idea is to start using `True16Predicate` gradually as will seem necessary for future patches.

Reply:
Do you have any examples that we can not use pasting to resolve the overriding issue for OtherPredicates?

Reply:
In real instructions we can paste with `OtherPredicates` of their pseudos, but there's no way to extend self `OtherPredicates` of classes and records. So every `let OtherPredicates = ...` potentially overrides some needed predicates and so having `True16Predicate` makes it a bit easier to avoid clashes.

Reply:
It is Okay to me to introduce True16Predicate if we don't know what is exactly in OtherPredicates when we try to extend it.